.eslintrc

```diff
@@ -2,7 +2,8 @@
   "plugins": ["prettier"],
   "extends": "eslint:recommended",
   "env": {
-	"node": true
+	"node": true,
+	"es6": true
```

Comment:
We support node 4 and up, and `Map` has been in node since 0.12, so we're good 🙂